en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/as-a-standalone-gateway.md

```diff
@@ -53,7 +53,7 @@ Following properties of the open API is not recommended to be edited, as it will
 - basePath - If swagger 2.0. version is used
 - x-wso2-basePath
 - [servers](https://swagger.io/docs/specification/api-host-and-base-path/) objects basePath section if `x-wso2-basePath` is not present in the open API (this is explained in the above section **How to work with standard open API definition** ).
-    For ex if servers URL is defined as below.
+    For an example if the server's URL is defined as below,
```

Comment:
Where is the example?

Reply:
Below yaml code snippet was there as the example. But in there, sentence was divided into two parts.  I'll change that.

Reply:
Fixed with: https://github.com/wso2/docs-apim/pull/5119/commits/ede4ebab54afe806430c80032c7b2541501206da